astropy/coordinates/erfa_astrom.py

```diff
@@ -238,6 +283,78 @@ def _get_cip(support, obstime):
             for cip_component in cip_support
         )
 
+    @staticmethod
+    def _get_era(support, obstime):
+        jd1_ut1_support, jd2_ut1_support = get_jd12(support, 'ut1')
+        era_support = erfa.era00(jd1_ut1_support, jd2_ut1_support)
+        return np.interp(obstime.mjd, support.mjd, era_support)
+
+    @staticmethod
+    def _get_polar_motion(support, obstime):
+        # TODO: do we need this? It's just a table lookup
+        polar_motion_support = get_polar_motion(support)
+        return tuple(
+            np.interp(obstime.mjd, support.mjd, polar_motion_component)
+            for polar_motion_component in polar_motion_support
+        )
+
+    @staticmethod
+    def _get_sp00(support, obstime):
+        # this is already done in _get_cip. Improve efficiency?
+        jd1_tt_support, jd2_tt_support = get_jd12(support, 'tt')
+        sp00_support = erfa.sp00(jd1_tt_support, jd2_tt_support)
+        return np.interp(obstime.mjd, support.mjd, sp00_support)
+
```

Comment:
I don't see the point of interpolating `erfa.era00()` and `erfa.sp00()` here. It would likely be faster not to do so as they are just a few lines of code. In the case of `erfa.sp00()`, only 2. Also the earth rotation angle is the one thing that changes the fastest here, so I really don't think linear interpolation is a good thing to do with that.

Reply:
Very true; it’s faster and more accurate to not interpolate - good catch.